tests/semver/test_constraint_utils.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+import poetry.semver
+from poetry.semver.constraint_utils import is_constraint
+from poetry.semver.constraint_utils import sorted_constraints
+
+# TODO: test PEP-0440 vs. SEM-VER constraints
+
+
+# from test_main.py
+@pytest.mark.parametrize(
+    "constraint,result",
+    [
+        ("*", True),
+        ("*.*", True),
+        ("v*.*", True),
+        ("*.x.*", True),
```

Comment:
i don't know these and they look strange to me.

Reply:
They are present elsewhere in poetry tests, this is a copy-n-paste and I don't know where they come from either.